corehq/apps/accounting/utils.py

```diff
@@ -124,6 +124,12 @@ def domain_has_privilege(domain, privilege_slug, **assignment):
             return False
         if plan_version.role.has_privilege(privilege):
             return True
+        # temporary until we feel safe changing roles - J$, Noah, Cal
+        elif (plan_version.role.slug != 'community_plan_v1' and
+              (privilege_slug == privileges.EXCEL_DASHBOARD or
+               privilege_slug == privileges.DAILY_SAVED_EXPORT)
+            ):
```

Comment:
E124 closing bracket does not match visual indentation
E125 continuation line with same indent as next logical line